# include general statistics for the camera

std_msgs/Header header
int32 exposure
int32 gain
int32 meanLux